tests/eq_qp_test.py

```diff
@@ -20,6 +20,7 @@
 import jax.numpy as jnp
 
 from jaxopt import projection
+from jaxopt._src.base import KKTSolution
```

Comment:
We should probably export `KKTSolution` [here](https://github.com/google/jaxopt/blob/main/jaxopt/base.py) so that we can do `from jaxopt.base import KKTSolution` (otherwise users don't have a way to set their own initialization).